NEWS.md

```diff
@@ -69,6 +69,8 @@
 * The `theme()` constructor now has named arguments rather than ellipsis. This 
   should make autocomplete substantially more useful.
 
+* geom_violin now again has a nicer looking range that allow the density to
```

Comment:
The range of each violin is now automatically extended 3 * bw for either end of the data range